.circleci/config.yml

```diff
@@ -261,6 +266,58 @@ jobs:
               export PATH=$HOME/miniconda/bin:/usr/local/cuda/bin:$PATH
               . activate habitat; cd habitat-sim
               python examples/example.py --scene data/scene_datasets/habitat-test-scenes/van-gogh-room.glb --silent --test_fps_regression $FPS_THRESHOLD
+      - add_ssh_keys:
+          fingerprints:
+            - "18:88:e0:37:b1:b3:7a:23:aa:1e:f7:43:a8:5b:8e:05"
+      - run:
+          name: Update documentation
+          command: |
+              export PATH=$HOME/miniconda/bin:/usr/local/cuda/bin:$PATH
+              . activate habitat;
+              cd habitat-sim
+              # Rebuild with all options enabled
+              ./build.sh --with-cuda --with-bullet
+              cd docs
+              conda install -y -c conda-forge doxygen
+              conda install -y  jinja2 pygments docutils
+              conda install -y -c conda-forge pelican
+              mkdir -p ../build/docs
+              git submodule update --init
+              doxygen
+              ./build-public.sh
+              cd ../..
+
+              # Update website
+              git clone git@github.com:facebookmicrosites/habitat-website.git
```

Comment:
This actual publish step should be run only once the PR is merged into master

Reply:
Yes, that's for testing purposes. I added it as incomplete steps in the PR description.